.prh.yaml

```
@@ -0,0 +1,129 @@
+version: 1
+rules:
+  - pattern: vald
+    expected: Vald
+    options:
+      wordBoundary: true
+    specs:
+      - from: vald
+        to: Vald
+      - from: valdcli
+        to: valdcli
+  - pattern: helm
+    expected: Helm
+    options:
+      wordBoundary: true
+  - pattern:
+      - Golang
+      - golang
+      - /go(?! to)/
+    expected: Go
+    options:
+      wordBoundary: true
+    specs:
+      - from: go
+        to: Go
+      - from: going
+        to: going
+      - from: go to
+        to: go to
+      - from: golang
+        to: Go
+      - from: algorithm
+        to: algorithm
+  - pattern: python
+    expected: Python
+    options:
+      wordBoundary: true
+  - pattern: clojure
+    expected: Clojure
+    options:
+      wordBoundary: true
+  - pattern: java
+    expected: Java
+    options:
+      wordBoundary: true
+  - pattern:
+      - node.js
+      - nodejs
+    expected: Node.js
+  - pattern:
+      - Yahoo Japan!
+      - Yahoo! Japan
+      - YJ
+    expected: Yahoo Japan
```

I think `Yahoo! Japan` is correct.
Ref: https://www.google.com/search?q=yahoo+japan&rlz=1C5CHFA_enJP938JP940&oq=yahoo+japan&aqs=chrome..69i57j0i512l2j69i60l3j69i65l2.3830j0j7&sourceid=chrome&ie=UTF-8

```suggestion
  - pattern:
      - Yahoo Japan!
      - Yahoo Japan
      - YJ
    expected: Yahoo! Japan
```